internal/scalibrextract/language/python/requirementsenhancable/requirementsenhanceable.go

```diff
@@ -17,12 +17,14 @@ const (
 
 // Extractor extracts python packages from requirements.txt files.
 type Extractor struct {
-	actual filesystem.Extractor
+	base     filesystem.Extractor
```

Comment:
nit: like with the pom extractor, I'd probably call this `online` and `offline` too